config/default/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-      - image: k8s.gcr.io/cluster-api/cluster-api-controller:v0.1.5
+      - image: us.gcr.io/k8s-artifacts-prod/cluster-api/cluster-api-controller:v0.1.6
```

Comment:
Fixing the prod repo, since the k8s.gcr.io vanity domain is not in place for the image promotion-based repository yet.